src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -55,6 +56,8 @@ public static int Main(string[] args)
 
             SetConnectionLimit();
 
+            SetUserAgent();
```

Comment:
We would need to set it for all tests. It would be nice to have that separated out from the online stats, but it adds that overhead all over.